docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs crontabber.
+
+# Run crontabber sleeping 5 minutes between runs
+while true; do
+    python socorro/cron/crontabber_app.py
+    echo "Sleep 5 minutes..."
+    sleep 300
+done
```

Comment:
This is a big behavior change from how crontabber runs currently.

Currently, it's kicked off every 5 minutes by cron. So if there's a crontabber running for longer than 5 minutes, a new one will get kicked off and they'll fight to the death and then things keep happening.

With this implementation, we wait for crontabber to finish running, then sleep 300 seconds, then run a new one. So instead of "runs every 5 minutes" it's now more like "runs and then sleeps for 5 minutes".

This is a stupid simple implementation, so that's why I did it this way for now. If the behavioral implications of this aren't ok, then we probably need to write a real scheduler or have it run crontabber in the background (will it capture stdout correctly?) or something like that.